src/main/resources/application.yml

```
@@ -3,6 +3,12 @@ spring:
       name: ${artifactId}
       version: ${version}
     jmx.enabled: true
+    datasource:
+      url: \${ORGANIZATION_DB_URL:jdbc:postgresql://\${ORGANIZATION_DB_HOST:localhost}:\${ORGANIZATION_DB_PORT:5432}/mypay}
```

let's make configurable also db name
```suggestion
      url: \${ORGANIZATION_DB_URL:jdbc:postgresql://\${ORGANIZATION_DB_HOST:localhost}:\${ORGANIZATION_DB_PORT:5432}/\${ORGANIZATION_DB_NAME:mypay}}
```